### PR TITLE
Fixes #76 - Swapping over to use package

### LINKED ISSUE #76: distuitils Version classes are deprecated
Hi @smarie!

Looks like a fairly simple change, but right now Im getting a ton of deprecation warnings:

```bash
generating gallery for /home/runner/work/ChainConsumer/ChainConsumer/docs/generated/gallery... :  80%|████████  | 4/5 [00:26<00:05,  5.45s/it]INFO    -  DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  File "/home/runner/work/ChainConsumer/ChainConsumer/.venv/lib/python3.10/site-packages/mkdocs_gallery/py_source_parser.py", line 109, in _get_docstring_and_rest
    if LooseVersion(sys.version) >= LooseVersion("3.7"):
  File "/home/runner/work/ChainConsumer/ChainConsumer/.venv/lib/python3.10/site-packages/setuptools/_distutils/version.py", line 55, in __init__
    warnings.warn(
```

Which boil down to:

`DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.`

Hopefully the fix is a simple swap here: https://github.com/smarie/mkdocs-gallery/blob/89fea9eacf6b9742fdbe2622f7d6a7d525db42a9/src/mkdocs_gallery/py_source_parser.py#L16

From

```python
from distutils.version import LooseVersion
```

to

```python
from packaging.version import Version
```

And then updating

```python
    if LooseVersion(sys.version) >= LooseVersion("3.7"):
```

I believe you could also make use of `from packaging.version import parse` if you wanted too.

### PR DESCRIPTION
**Pull request by Samreay**
Fixes #76

### CODE REVIEW

**Comment by smarie** [comment]
Thanks @Samreay !
Could you please simply pull latest from develop and add a changelog entry ? Otherwise all good

**Comment by Samreay** [comment]
@smarie changelog entry added

File: docs/changelog.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### 0.7.9 - (In Progress)
 
- - TBD
+ - Swapped from deprecated `disutils.version` to `packaging.version`. Fixes [#76](https://github.com/smarie/mkdocs-gallery/issues/76)
```
Comment (smarie) [inline docs/changelog.md:5]:
```suggestion
 - Swapped from deprecated `disutils.version` to `packaging.version`. PR [#79](https://github.com/smarie/mkdocs-gallery/pull/79) by [Samreay](https://github.com/Samreay)
```

**Comment by smarie** [comment]
All good @Samreay thanks for this PR !

**Pull request merged**